commands/FBDebugCommands.py

```diff
@@ -189,6 +191,58 @@ def run(self, arguments, options):
     fb.evaluateEffect('[[UIApplication sharedApplication] performSelector:@selector(_performMemoryWarning)]')
 
 
+def switchBreakpointState(expression,on):
+  ci = lldb.debugger.GetCommandInterpreter()
+  res = lldb.SBCommandReturnObject()
+
+  ci.HandleCommand('br list',res)
+  listStr = res.GetOutput()
+
+  br_list = listStr.splitlines()
+  expression_pattern = re.compile(r'{}'.format(expression),re.I)
+  id_pattern = re.compile(r'^\s*([1-9]\d*)(\.\d+)?',re.I)
```

Comment:
What do you think about using the Python LLDB API, instead of parsing the output of `breakpoint list`?

For example, it would roughly look like:

```python
for breakpoint in target.breakpoint_iter():
  for location in breakpoint:
    if pattern.search(location.GetAddress().symbol.name):
      location.SetEnabled(False)
```

Reply:
```
  target = lldb.debugger.GetSelectedTarget()
  for breakpoint in target.breakpoint_iter():
    for location in breakpoint:
      if expression_pattern.search('{}'.format(location)):
        location.SetEnabled(on)
```
What do you think about using location? 
They have some different.
```
location	2.1: where = SunLigth`__66+[SUNNetService httpAsyncPostWithDictionary:completionBlock:]_block_invoke + 1984 at SUNNetService.m:37, address = 0x0000000104514dfc, unresolved, hit count = 0  Options: disabled 
address		SunLigth`__66+[SUNNetService httpAsyncPostWithDictionary:completionBlock:]_block_invoke + 1984 at SUNNetService.m:37
symbol		id = {0x00006db1}, range = [0x00000001001e063c-0x00000001001e0ec8), mangled="__66+[SUNNetService httpAsyncPostWithDictionary:completionBlock:]_block_invoke"
name 		__66+[SUNNetService httpAsyncPostWithDictionary:completionBlock:]_block_invoke

```
So that,we can use `benable disabled` to switch all breakpoints to `enable`,and also as follow
```

#use `benable disabled` to switch all breakpoints to `enable`
benable disabled

* benable ***address***
benable 0x0000000104514dfc

* benable ***filename***
benable SUNNetService.m 

* benable ***module***
benable UIKit
benable SunLigth 

```

Reply:
Enabling/disabling via filename and module definitely seem useful!